google/cloud/forseti/services/cli.py

```diff
@@ -15,6 +15,7 @@
 """Forseti CLI."""
 
 from argparse import ArgumentParser
+import grpc
```

Comment:
This is not a standard library, and should be move down below, and separated from the standard libraries by a newline.